src/styles/block/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+
+  &__img {
+    margin-left: 3px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin-top: 36px;
+    margin-bottom: 4px;
```

Comment:
It's better to use one style of vertical margin: either margin-top or margin-bottom. Try to move one of these to the next or previous element